src/vmm/src/vstate/memory.rs

```diff
@@ -32,23 +32,23 @@ pub type GuestMmapRegion = vm_memory::MmapRegion<Option<AtomicBitmap>>;
 /// Errors associated with dumping guest memory to file.
 #[derive(Debug, thiserror::Error, displaydoc::Display)]
 pub enum MemoryError {
-    /// Cannot access file: {0:?}
+    /// Cannot access file: {0}
     FileError(std::io::Error),
-    /// Cannot create memory: {0:?}
+    /// Cannot create memory: {0}
     CreateMemory(VmMemoryError),
-    /// Cannot create memory region: {0:?}
+    /// Cannot create memory region: {0}
     CreateRegion(MmapRegionError),
-    /// Cannot fetch system's page size: {0:?}
+    /// Cannot fetch system's page size: {0}
     PageSize(errno::Error),
-    /// Cannot dump memory: {0:?}
+    /// Cannot dump memory: {0}
     WriteMemory(GuestMemoryError),
     /// Cannot create mmap region: {0}
     MmapRegionError(MmapRegionError),
     /// Cannot create guest memory: {0}
     VmMemoryError(VmMemoryError),
     /// Cannot create memfd: {0:?}
```

Comment:
Was this missed?
```suggestion
    /// Cannot create memfd: {0}
```

Reply:
Mh, true. `memfd::Error` does implement `Display` (https://docs.rs/memfd/latest/src/memfd/errors.rs.html#23-31), so this would be nice to update